### DISCUSSION TITLE
_sa as dict key

### DISCUSSION THREAD

**Discussion by Omarsawan** [fastapi/fastapi#9959] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
https://github.com/tiangolo/fastapi/blob/8d2723664801c32e6174cd87812fb92e850cfa4b/fastapi/encoders.py#L182
```

### Description

I have an API that returns a dictionary as HTTP response, one of my keys starts with '_sa', and I found this line of code here
https://github.com/tiangolo/fastapi/blob/8d2723664801c32e6174cd87812fb92e850cfa4b/fastapi/encoders.py#L182,
which ignores keys that starts with '_sa', how can I avoid ignoring the keys because they are needed to be returned in the response?
Thank you

### Operating System

macOS

### FastAPI Version

0.89.1

### Python Version

3.11

**Comment by jgould22** [comment]
That is frustrating.

An immediate work around would be use [ORJSONResponse](https://fastapi.tiangolo.com/advanced/custom-response/#use-orjsonresponse) like below, but you need to make sure that everything in your dict is json serializable 

```python
from fastapi import FastAPI
from fastapi.responses import ORJSONResponse

app = FastAPI()
@app.get("/")
def home():
    return ORJSONResponse([{"test":"test","_sa_test":"_sa_test"}])
```

**Reply by Kludex** [reply, marked as answer]
That line of code will soon be removed when SQLModel supports pydantic v2.

**Comment by Omarsawan** [comment]
Thank you
